src/pages/task/taskTabs/testsTable/LogsColumn.tsx

```diff
@@ -1,65 +1,24 @@
 import styled from "@emotion/styled";
 import Button from "@leafygreen-ui/button";
 import { Link } from "react-router-dom";
-import { Analytics } from "analytics/addPageAction";
+import { Analytics } from "analytics/task/useTaskAnalytics";
 import { getTaskHistoryRoute } from "constants/routes";
 import { size } from "constants/tokens";
 import { TestResult, GetTaskQuery } from "gql/generated/types";
 import { TestStatus } from "types/test";
 import { string } from "utils";
-import {
-  getLobsterURL,
-  getParsleyUrl,
-  isProduction,
-} from "utils/environmentalVariables";
-import { stringifyQuery } from "utils/queryString";
 
 const { escapeRegex } = string;
 interface Props {
-  taskAnalytics: Analytics<
-    | { name: "Click Logs Lobster Button" }
-    | { name: "Click Logs HTML Button" }
-    | { name: "Click Logs Raw Button" }
-    | { name: "Click See History Button" }
-  >;
+  onClick: Analytics["sendEvent"];
```

Comment:
nit. naming wise, I think it would make more sense to call this function `sendEvent`, since by its type that is all it would ever do. (Also would be more obvious what it is doing below when we call it)

Reply:
Removed it all together